lib/modules/manager/bazel-module/bazelrc.ts

```diff
@@ -96,6 +96,36 @@ function createEntry(line: string): BazelrcEntries | undefined {
   return undefined;
 }
 
+export function expandWorkspacePath(
+  value: string,
+  workspaceDir: string,
+): string {
+  if (!value.includes('%workspace%')) {
+    return value;
+  }
+  const absolutePath = upath.resolve(workspaceDir);
+  const expandedPath = value.replace('%workspace%', absolutePath);
+  if (!fs.isValidLocalPath(expandedPath)) {
+    throw new Error(
```

Comment:
Should this really be a thrown error? Will it abort the entire renovate run?

Reply:
Being a bit defensive here in coding, if file doesnt exist it should be a problem. Would you recommend removing this?

Reply:
I think the dependency should just be skipped, instead of ruining the whole run